server/datastore/mysql/migrations/tables/20230807100822_AddIndexNanoEnrollment_test.go

```diff
@@ -0,0 +1,20 @@
+package tables
```

Comment:
can we remove this file since we're not adding any tests? seems like we're doing that for all migrations that only add indexes.

optionally we could add a function similar to this one so we can start testing indexes, but not required by any means.

https://github.com/fleetdm/fleet/blob/b4e3d9d6774bde46c35a244677dbb0ad252f320f/server/datastore/mysql/migrations/tables/migration.go#L14-L28